Kitodo/src/main/webapp/WEB-INF/resources/css/kitodo.css

```diff
@@ -441,7 +422,151 @@ main {
     height: 500px;
     }
 
+#listWrapper {
+    width: 95%;
+    margin: auto;
+    border-radius: 5px;
+    padding: 0;
+}
+
+#listWrapper > div {
+    padding: 0;
+}
+
+.content-header {
+    height: 30px;
+    padding: 1.3em 1.5em;
+}
+
+.content-header .ui-panel-content {
+    padding: 0;
+}
+
+.content-header .ui-panel-content > h3 {
+    display: inline;
+    line-height: 30px;
+}
+
+.content-header .ui-panel-content > form,
+.content-header .ui-panel-content > button {
+    float: right;
+    margin-left: 20px;
+}
+
+.input {
+    width:100%;
+}
+
+.input-with-button,
+.input-with-button > input {
+    width: calc(100% - 44px); !important;
+}
+
+.input-with-button + .ui-button,
+.input-with-button > input + .ui-button {
+    margin-left: 5px;
+    margin-right: 0;
+}
+
+.input-with-button.ui-calendar > button {
+    position: relative;
+}
+
+.input-with-button.ui-calendar > button > .ui-icon-calendar {
+    background: transparent;
+}
+
+.input-with-button.ui-calendar > button::before {
+    content: '\f073';
+    font-family: 'FontAwesome';
+    font-weight: lighter;
+    position: absolute;
+    left: 0;
+    padding: .7em;
+}
+
+.filter-input {
+    width: 250px;
+    border-radius: 3px 0 0 3px !important;
+}
+
+.filter-menu {
+    border-radius: 3px;
+}
+
+.filter-menu > div {
+    padding: 0;
+    width: 250px;
+    background: #f3f3f3;
+    border-radius: 3px;
+}
+
+.filter-menu li {
+    list-style: none;
+    border-top: solid 1px #d8d8d8;
+}
+
+.filter-menu li a {
+    display: inline-block;
+    padding: 10px;
+    text-decoration: none;
+    color: #006DBD;
+}
+
+.filter-menu li > a:first-child {
+    width: 75%;
+}
+
+.filter-menu li:first-child a {
+    width: 100%;
+    box-sizing: border-box;
+    border-top-left-radius: 3px;
+    border-top-right-radius: 3px;
+}
+
+.filter-menu li .filter-delete {
+    float: right;
+    vertical-align: middle;
+}
+
+.ui-tabs .ui-tabs-panel > div.mets-file-groups,
+.ui-tabs .ui-tabs-panel > div.mets-file-groups > div {
+    padding: 0;
+}
+
+.mets-file-groups .ui-datatable table {
+    border-top: solid 1px #d8d8d8;
+}
 
+.mets-file-groups .ui-datatable.ui-widget tr th,
+.mets-file-groups .ui-datatable.ui-widget tr td {
+    padding: .75em 1.5em;
+}
+
+.mets-file-groups .ui-datatable.ui-widget tr th:last-child,
+.mets-file-groups .ui-datatable.ui-widget tr td:last-child {
+    width: 100px;
+    text-align: right;
+}
+
+.mets-file-groups .ui-datatable.ui-widget tr th {
+    background: transparent;
+}
+
+
+
+.mets-file-groups > div > button {
+    margin: 1.5em;
+}
+
+
+/*.input {
```

Comment:
If this CSS instructions are not used, then remove them. If they are needed later, add them later.